Gemfile.lock

```diff
@@ -106,7 +106,7 @@ GEM
       railties (>= 5.0.0)
     faker (2.14.0)
       i18n (>= 1.6, < 2)
-    ffi (1.13.1)
+    ffi (1.14.2)
```

Comment:
I did not change the `ffi` version explicitly. I just ran `bundle` and this was the resulting `Gemfile.lock`.